libs/forms/src/FormItemElement.ts

```diff
@@ -36,6 +36,28 @@ export abstract class FormItemElement extends UpdatingElement {
   @property()
   label: string;
 
+  /**
+   * @property
+   * @kind boolean
```

Comment:
注释里的 `@property` 是多余的，下同。

类型有错：

```suggestion
   * @kind string
```